app/src/androidTest/java/fr/free/nrw/commons/AboutActivityTest.kt

```diff
@@ -0,0 +1,116 @@
+package fr.free.nrw.commons
+
+import android.app.Activity
+import android.app.Instrumentation
+import android.content.Intent
+import androidx.test.core.app.ApplicationProvider.getApplicationContext
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
+import fr.free.nrw.commons.utils.ConfigUtils
+import org.hamcrest.CoreMatchers
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class AboutActivityTest {
+    @get:Rule
+    var activityRule: ActivityTestRule<*> = ActivityTestRule(AboutActivity::class.java)
```

Comment:
This rule should just launch you to `AboutActivity`, why do you have to launch the app from the beginning?

Reply:
@macgills Thanks for pointing it out, I thought it was necessary to skip the welcome screen and login screen.